test/nodes/grovers_spec.js

```diff
@@ -32,22 +30,22 @@ describe('GroversNode', function() {
   it('return success output on valid input', function(done) {
     let flow = [{id: 'groversNode', type: 'grovers', wires: [['helperNode']]},
       {id: 'helperNode', type: 'helper'}];
+
     nodeTestHelper.load(groversNode, flow, function() {
       let groversTestNode = nodeTestHelper.getNode('groversNode');
       let helperNode = nodeTestHelper.getNode('helperNode');
+
       helperNode.on('input', function(msg) {
         const expectedPayload = 'Top measurement: 111111\n' + 'iterations = 6';
         try {
-          msg.should.have.property('payload', expectedPayload);
+          assert.strictEqual(msg.payload, expectedPayload);
           done();
         } catch (err) {
           done(err);
         }
-        finally {
-          groversTestNode.shell.stop();
-        }
       });
+
       groversTestNode.receive({payload: '111111'});
     });
-  }).timeout(10000);
```

Comment:
this timeout is needed because, on average it takes 5-7 seconds to run this test.

Reply:
Are you sure? The CI is coming up as only 1364ms:
`return success output on valid input (1364ms)`

Reply:
Let's leave as it is for now, but if in future it fails that we will know what's problem